### ISSUE TITLE
Blue Screen in nipcibrd.sys on boot after installing game tools

### ISSUE THREAD

**Issue by sciencewhiz** [wpilibsuite/2024Beta#7]
I get a BSOD: nipcibrd.sys SECURE_PCI_CONFIG_SPACE_ACCESS_VIOLATION after installing game tools. My laptop is a 13th gen Intel 1340p with Windows 11 pro. Virtualization based security is enabled.

This is the convoluted workaround I followed to get my computer usable again (inspired by https://forums.ni.com/t5/LabVIEW/How-to-disable-the-nipcibrd-sys-driver-to-prevent-hibernation/m-p/4296535). More context: https://forums.ni.com/t5/LabVIEW/BSOD-nipcibrd-sys-SECURE-PCI-CONFIG-SPACE-ACCESS-VIOLATION/td-p/4284462 and https://forums.ni.com/t5/General/Why-does-Game-Tools-require-PXI/td-p/4315440

1. Boot into the bios and disable virtualization extensions
2. Download the offline installer from: https://www.ni.com/en-us/support/downloads/drivers/download.pxi-platform-services.html#479592 for 2023 q2)
3. Mount the downloaded iso
4. Navigate to the pool directory
5. using 7-zip, open ni-pcibrd_22.5.0.49208-0+f56_windows_x64.nipkg (version may be different)
6. double click on data.tar.gz
7. double click on data.tar
8. double click on the . folder
9. extract the 5 files, which includes pcibrdi32.msi and pcibrdi64.msi
10. Run both msi's, choosing the uninstall option.
11. reboot and re-enable virtualization extensions

**Comment by sciencewhiz** [comment]
Upgrading to PXI Platform Services 2023 Q4 (released last week) also solved the BSOD for me. But I haven't check to see if there's any side effects of that.

**Comment by EthyMoney** [comment]
Lenovo laptop by chance? I had this issue with Lenovo laptops with "Enhanced Windows Biometric Security" enabled in the BIOS. Once that got off and it was good again.

**Comment by sciencewhiz** [comment]
No, it's a Framework 13. But based on googling, Lenovo's Enhanced Windows Biometric Security appears to enable virtualization based security, so it looks to be the root cause.

**Comment by PeterJohnson** [comment]
Uninstalling this driver also appears to fix crash-on-wake issues I was having with my Windows 11 install on an Ryzen 3rd gen desktop machine.  I tried 2023Q4, but that didn't fix the issue.
